kafka-rest/src/main/java/io/confluent/kafkarest/resources/AsyncResponses.java

```diff
@@ -104,8 +108,10 @@ public void asyncResume(AsyncResponse asyncResponse) {
                 asyncResponse.resume(responseBuilder.entity(entity).build());
               }
             } else if (exception instanceof CompletionException) {
+              log.error("AsyncResponse CompletionException: {}", exception);
```

Comment:
nit: I am not sure if we would be able to link the response error to the request to which it corresponds. Could we log something additional from the async response to achieve that?

Reply:
Yes - but the AsyncResponse doesn't have anything useful in it, and the entity is a generic type that extends Object, so I cant guarantee anything more than the class, hash and hope that there is a useful toString defined on the object.